search-sentiments-map/public/index.html

```diff
@@ -9,14 +9,46 @@
     <!-- Bootstrap CSS. -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
-
-    <script src="script.js"></script>
+    <script src="scripts/script.js"></script>
+    <script src="scripts/mapScript.js"></script>
```

Comment:
According to the style guide: Javascript file names must be all lowercase and may include underscores (_) or dashes (-). Let's all do dash for this project to be consistent.